### ISSUE TITLE
[en] false alarms EN_UNPAIRED_BRACKETS

### ISSUE THREAD

**Issue by tiff** [languagetool-org/languagetool#2356]
> They won sixteen championships (seven constructors' and nine drivers') in that period, while Williams used engines from Ford, Honda, and Renault to also win sixteen titles (nine constructors' and seven drivers').

<img width="891" alt="Bildschirmfoto 2020-01-22 um 12 02 06" src="https://user-images.githubusercontent.com/37363/72889025-18e9ca00-3d0f-11ea-9a3e-352c348ecba8.png">

> We're goin' home.

<img width="443" alt="Bildschirmfoto 2020-01-22 um 12 03 07" src="https://user-images.githubusercontent.com/37363/72889066-2a32d680-3d0f-11ea-9251-106591b7b1ca.png">

**Comment by danielnaber** [comment]
I've made the rule less strict, let's see whether that helps with the false alarms (only affects the English rule).

**Comment by danielnaber** [comment]
It seems the change helped quite a bit:

![Bildschirmfoto von 2020-02-06 12-04-21](https://user-images.githubusercontent.com/857004/73931571-d6acb500-48d8-11ea-9c9a-67b912dace7d.png)

**Comment by gavinbunney** [comment]
@danielnaber We are hitting issues with this rule for the less strict improvements, which for sure helps with single quotes `'`, but is a bit unexpected for double quotes.

Example:
```
Click the "Check text button
```

Expect a warning on the `"` saying it doesn't have a matched pair, however it is not. Can replicate this on the public languagetool.org editor as well.

Does it makes sense to reintroduce the unpaired quote matcher for double-quotes?

**Comment by danielnaber** [comment]
@gavinbunney Yes, I'll give it a try. I have committed a change to check for double quotes again and I'll have a look at the user acceptance numbers the day after tomorrow.

**Comment by danielnaber** [comment]
@gavinbunney I'm sorry I had to roll back my change, as there are new false alarms and I lack the time to analyze them now.

For example, this causes a false alarm:

> Its remains are known today as Kastell Aalen (”Aalen roman fort“).

**Comment by MikeUnwalla** [comment]
Also:

> I want to hide the folder from the git view, so that I **a)** don’t accidentally commit it and **b)** don't ...
